src/components/App/App.js

```diff
@@ -33,17 +45,18 @@ class App extends Component {
           <Errors statusCode={match.params.code}/>
         } />
 
-        <Route exact path='/' render={ () => 
+        <Route exact path="/" render={ () => 
           this.state.isLoading ? <div className="loader"></div> : 
           !this.state.response ? <Redirect to={`/error/${this.state.statusCode}`} /> :
-          <PosterGrid movies={this.state.movieData}/> } 
-          />
+          <>
+            <Searchbar updateSearch={this.updateSearch} />
+            <PosterGrid movies={this.state.useSearch ? this.state.searchData : this.state.movieData}/>  
+          </>
+        } />
 
-        
         <Route exact path="/:movieID" render={({match}) => 
-            <Movie movieID={match.params.movieID}/> 
-          }
-        />
+          <Movie movieID={match.params.movieID}/> 
+        } />
```

Comment:
Do you want to move to this format for everything? That is fine by me, just want to make sure we go through together and match it throughout the project.